### PR TITLE
LIME-139: Create notifications component.

### LINKED ISSUE #139: Create notification component in header
Image

Use this component from LMS as reference
When we click the icon we should see list of notifications

You can mock the data for now

### PR DESCRIPTION
**Pull request by DavidCharif**
image
image
image

https://github.com/BinaryStudioAcademy/bsa-winter-2023-2024-lime/assets/53876827/156179ba-b239-4a52-8a5e-0e602b4f433a

### CODE REVIEW

**Comment by nikita-remeslov** [comment]
Looks like it does not fit our design. we should update the colors

**Comment by DavidCharif** [comment]
> Looks like it does not fit our design. we should update the colors



https://github.com/BinaryStudioAcademy/bsa-winter-2023-2024-lime/assets/53876827/1667fd27-1722-4038-a78e-a7c9470969ca

image
image
image

**Comment by ArtemLymarenko** [comment]
You can try to use popover component from common folder and not create all those divs with position relative/absolute.

**Comment by DavidCharif** [comment]
https://github.com/BinaryStudioAcademy/bsa-winter-2023-2024-lime/assets/53876827/904559d7-6ad8-4597-911a-7a8bb8dd7335

image

**Review by dimapopovych — DISMISSED** [review]

File: frontend/src/bundles/common/components/header/components/notifications/enums/notifications.enums.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+type NotificacitionType = {
```
Comment (dimapopovych) [inline frontend/src/bundles/common/components/header/components/notifications/enums/notifications.enums.ts]: this is not enum, move to proper folder

File: frontend/src/bundles/common/components/header/components/notifications/notification-component.tsx [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    useHandleClickOutside,
+    useRef,
+    useState,
+} from '~/bundles/common/hooks/hooks.js';
+
+import { NotificationList } from './notification-list.js';
+import { NotificationIcon } from './notifications-header.js';
+
+const NotificationComponent = (): JSX.Element => {
+    const [notifications, setNotifications] = useState([
```
Comment (dimapopovych) [inline frontend/src/bundles/common/components/header/components/notifications/notification-component.tsx]: maybe we can move this state to the store and add necessary actions as well at this point?

**Review by dimapopovych — CHANGES_REQUESTED** [review]

**Comment by nikita-remeslov** [comment]
many changes not related to you task

**Review by nikita-remeslov — CHANGES_REQUESTED** [review]
the same. you did not change strave in your ticket. You should fix the diff of your PR

**Pull request closed without merging**